storage-resize-images/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Version 0.1.8
+
+fixed - fix bug where resized image did not keep orientation of original image (Issue #290
+fixed - release automation script for Github release (#302)
+fixed - clear cache so that image names can be reused (Issue #286) (#297)
+fixed - switched ImageMagick for Sharp library (#207)
```

Comment:
These two items are also listed in the previous release.  I think that we need some clarity about what exactly was changed to have the same fixes mentioned again.

Reply:
@laurenzlong can you provide more context here?

Reply:
only the first line should be in this release.

Reply:
When you say "first line"... do you mean that only the first bullet (the one about orientation) should be in this release?

Reply:
yes.